src/CMakeLists.txt

```diff
@@ -192,22 +192,40 @@ add_executable(slade WIN32 MACOSX_BUNDLE
 	${SLADE_HEADERS}
 )
 
-target_link_libraries(slade
-	${ZLIB_LIBRARY}
-	${BZIP2_LIBRARIES}
-	${EXTERNAL_LIBRARIES}
-	${wxWidgets_LIBRARIES}
-	${FREEIMAGE_LIBRARIES}
-	${SFML_LIBRARY}
-	${FTGL_LIBRARIES}
-	${OPENGL_LIBRARIES}
-	${GLEW_LIBRARY}
-	${CURL_LIBRARIES}
-	${LUA_LIBRARIES}
-	${MPG123_LIBRARIES}
-	${fmt_LIBRARIES}
-	-lstdc++fs
-)
+if(APPLE) 
+	target_link_libraries(slade
```

Comment:
`target_link_libraries()` can be used several times with the same target. Please restore the initial version without `stdc++fs`, and add it for Linux only with a separate call.

Actually, it should be added on any platform if we link with stdc++ standard library implementation.
However, proper detection of filesystem support in general requires a [separate module](https://github.com/vector-of-bool/CMakeCM/blob/6cb8a6ada09eb40c271d2f83a032e03cd4005fa3/modules/FindFilesystem.cmake).

Reply:
Done